apps/bas/src/screens/InsertCardScreen.tsx

```diff
@@ -26,7 +26,7 @@ const InsertCardScreen: React.FC<Props> = ({ lockScreen }) => {
               src="/images/insert-card.svg"
               alt="Insert Card Diagram"
             />
-            <h1>Insert Voter Card</h1>
+            <h1>Insert a New Voter Card</h1>
```

Comment:
```suggestion
            <h1>Insert a Voter Card</h1>
```

Reply:
Because I think that "New" will be confused with "old" or "used" voter cards.

Reply:
Updated copy, but didn't update the screenshot.